pkg/auth/iam/iam.go

```diff
@@ -41,8 +41,8 @@ type Filter struct {
 
 // ErrorResponse is the generic structure for communicating errors from a REST endpoint.
 type ErrorResponse struct {
-	ErrorCode    int `json:",omitempty"`
-	ErrorMessage string
+	ErrorCode    int    `json:"errorCode,omitempty"`
```

Comment:
why omitempty on the code?

Reply:
I thought this is intended for some cases, so I left it as it is. Should we remove this omitempty? Since the errorCode is required in other service, example: IAM

Reply:
update from offline discussion: since errorCode and errorMessage have become the standard for error Response, I will delete the omitempty for this. I will update this shortly. Thanks.